Source/web/web.gypi

```diff
@@ -320,6 +320,12 @@
       'WebMediaPlayerClientImpl.cpp',
       'WebMediaPlayerClientImpl.h',
     ],
+    'web_files_mediastream': [
```

Comment:
I don't find this "web_files_mediastream" be referenced anywhere.  I guess it should be used in "web.gyp"?